packages/kit/src/views/Overlay/UpdateItem/index.desktop.tsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import { FC } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import {
+  Box,
+  Divider,
+  Icon,
+  Text,
+  useIsVerticalLayout,
+} from '@onekeyhq/components';
+import PressableItem from '@onekeyhq/components/src/Pressable/PressableItem';
+import { useAutoUpdate } from '@onekeyhq/kit/src/hooks/redux';
+import { useCheckUpdate } from '@onekeyhq/kit/src/hooks/useCheckUpdate';
+import platformEnv from '@onekeyhq/shared/src/platformEnv';
+
+const UpdateItem: FC = () => {
+  const intl = useIntl();
+  const isVerticalLayout = useIsVerticalLayout();
+  const { state } = useAutoUpdate();
+  const { showUpdateBadge } = useCheckUpdate();
+
+  let formText = '';
+  const disabled = state === 'downloading';
+  if (state === 'available') {
+    formText = intl.formatMessage({ id: 'action__update_available' });
+  } else if (state === 'ready') {
+    formText = intl.formatMessage({ id: 'action__update_now' });
+  }
+
+  if (!formText) {
+    return null;
+  }
+
+  if (!platformEnv.isDesktop) return null;
+
+  return (
+    <>
+      <Divider />
+      <PressableItem
+        flexDirection="row"
+        alignItems="center"
+        justifyContent="space-between"
+        py={{ base: '12px', sm: '8px' }}
+        px={{ base: '16px', sm: '8px' }}
+        mt="4px"
+        bg="transparent"
+        borderRadius="12px"
+        onPress={() => {
+          if (state === 'ready') {
+            window.desktopApi.installUpdate();
+          } else {
+            window.desktopApi.downloadUpdate();
+          }
```

Comment:
```suggestion
          if (state === 'ready') {
            window.desktopApi?.installUpdate?.();
          } else {
            window.desktopApi?.downloadUpdate?.();
          }
```

这里改了一下，用 optional chaining 避免 desktopApi 在 runtime undefined 的情况

Reply:
啊原来是suggested change啊，我以为你直接push了，这里会有undefined的情况吗？我在后面的pr补上吧